### PR TITLE
feat(*): adds navigation

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: packages/ui-library/src/components/navigation/help-menu.tsx
Comment (th0rOdinson) [inline packages/ui-library/src/components/navigation/help-menu.tsx:1]: Seems to be empty

File: apps/root/src/frame/components/navigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  const currentRoute = useCurrentRoute();
+  const intl = useIntl();
+  const activeWallet = useActiveWallet();
+  const mode = useThemeMode();
+  const selectedLanguage = useSelectedLanguage();
+  const changeLanguage = useChangeLanguage();
+
+  React.useEffect(() => {
+    if (location.pathname === '/positions') {
+      dispatch(changeRoute('positions'));
```
Comment (th0rOdinson) [inline apps/root/src/frame/components/navigation/index.tsx:76]: I think we should decouple routes from tabs reducer, using `react-roter-dom` functions. We would simplify every redirect with just one operation. And hooks like `useCurrentRoute could just be replaced by implementing `useLocation`. What do you think?

**Review by th0rOdinson — COMMENTED** [review]
This looks sooooo good 🚀

File: apps/root/src/frame/components/navigation/index.tsx (hunk 1 above)
Comment (FiboApe) [reply apps/root/src/frame/components/navigation/index.tsx:76]: good idea

**Review by th0rOdinson — APPROVED** [review]

**Pull request merged**